packages/web-runtime/src/components/SidebarNav/SidebarNav.vue

```diff
@@ -72,28 +87,19 @@ export default {
       resizeObserver.disconnect()
     })
   },
-  computed: {
-    ...mapState(['navigation']),
-
-    toggleSidebarButtonClass() {
-      return this.navigation.closed
-        ? 'toggle-sidebar-button-collapsed'
-        : 'toggle-sidebar-button-expanded oc-pr-s'
-    },
-
-    toggleSidebarButtonIcon() {
-      return this.navigation.closed ? 'arrow-drop-right' : 'arrow-drop-left'
-    }
-  },
   methods: {
     ...mapActions(['openNavigation', 'closeNavigation']),
+    ...mapActions('Files', ['resetFileSelection']),
```

Comment:
We can't introduce a dependency on the files app from the runtime, since it's not required to load the files app at all (yes, we do have a production use case for that). Apart from that I'm not convinced that focussing the left sidebar should result in clearing the file selection. 🤔 What's the reasoning behind this?